picard/coverart/providers/__init__.py

```diff
@@ -61,33 +60,20 @@ def register_cover_art_provider(provider):
 
 # named tuples used by cover_art_providers()
 ProviderTuple = namedtuple('ProviderTuple', 'name title enabled cls')
-PInfoTuple = namedtuple('PInfoTuple', 'position enabled')
-POrderTuple = namedtuple('OrderTuple', 'name position enabled')
 
 
 def cover_art_providers():
-    def from_ca_providers_option():
-        """Iterate through ca_providers option and yield name, position and enabled"""
-        config = get_config()
-        for pos, (name, enabled) in enumerate(config.setting['ca_providers']):
-            yield POrderTuple(name=name, position=pos, enabled=enabled)
+    config = get_config()
+    PInfoTuple = namedtuple('PInfoTuple', 'position enabled')
```

Comment:
Why move it here? Wouldn't it be better to define this type only once?

Reply:
I agree, not a good move, reverted, and rebased.